content/2019-06-02-newsletter-18.md

```diff
@@ -1,6 +1,6 @@
 +++
 title = "The Embedded Working Group Newsletter - 18"
-date = 2019-04-05
+date = 2019-06-02
```

Comment:
```suggestion
date = 2019-07-02
```
Similarly the filename should be moved a month. ;)

Reply:
... wow. I probably shouldn't have done this late at night :)